source/history.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var likely = require('./index.js');
+var utils = require('./utils');
+
+/**
+ * History
+ * @type {Object}
+ */
+var history = {
+    /**
+     * Called everytime the page url's been changed.
+     * Reinits all widgets with the new page url.
+     * @type {Function}
+     */
+    onUrlChanges: function () {
```

Comment:
Also, missed this. What do you think about `onUrlChanges` vs `onUrlChange`? I like the latter more as it’s like native `onChange`.